.travis.yml

```diff
@@ -40,6 +40,7 @@ matrix:
       env: TOXENV=py37-upgrade
 before_install:
   - .travis/before_install.sh
+  - sudo sh -c "echo 1 > /proc/sys/kernel/unprivileged_userns_clone"
```

Comment:
Isn't this duplicate with the setting in install_podman.yml?

Reply:
The problem is that the setting in install_podman.yml won't work from within the molecule container, which is why the Travis tests are red.
I was hoping setting it before the molecule run, would make the tests turn green, since there would be nothing left to do during the test run. However, this is obviously not the case. I will look into how to repair the tests (ideally without this line) tomorrow.

Reply:
With the switch to GitHub actions this line is gone for now.
However something similar may be needed for the GitHub actions tests...

Reply:
you can add it here: https://github.com/pulp/ansible-pulp/blob/master/.github/workflows/ci.yaml#L37
(after validating commit) 
```yaml
- name: Before Install
  run: sudo sh -c "echo 1 > /proc/sys/kernel/unprivileged_userns_clone"
```

Reply:
I tried it, but it does not appear to make a difference.
For now I just left the failing Debian task with `ignore_errors: true`.
I am out of ideas what else to do about it.